Tests/CreateAPITests/GenerateOptionsTests.swift

```diff
@@ -660,5 +660,25 @@ final class GenerateOptionsTests: GenerateBaseTests {
         
         // THEN
         try compare(package: "strip-parent-name-nested-objects-default")
-    }        
+    }
+    
+    func testPestoreIdentifiableEnabled() throws {
+        // GIVEN
+        let command = try Generate.parse([
+            pathForSpec(named: "petstore", ext: "yaml"),
+            "--output", temp.url.path,
+            "--package", "petstore-only-schemas",
+            "--generate", "entities",
+            "--config", config("""
+            entities:
+                autogenerateIdentifiableConfomance: true
```

Comment:
Could we also include the renaming in the example just to make sure that we have some coverage of that also

```suggestion
            entities:
                autogenerateIdentifiableConfomance: true
            rename:
                properties:
                    Error.code: id
```